core/callitopservice.class.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class CallItopService
```

Comment:
Please add a `@since 1.3.0` so people extending the collector know since when this is available if they want to use the service directly.